### PR TITLE
Feature/pptt 3516 iframe support in form analytics

### PR DESCRIPTION
**Pull request by RomanSek**

### CODE REVIEW

**Review by kbigos — APPROVED** [review]
The code looks good (didn't test it)

File: templates.js [hunk 1]
```diff
@@ -439,6 +449,13 @@ fa.sendEvent(formAnalytics.event.{{eventName}});
             'Example: `{"name": "First name", "surname": "Last name"}`.',
           default: '{}'
         },
+        {
+          id: 'iframeTracking',
+          type: 'boolean',
+          displayName: 'Send messages from iframes',
+          description: 'If checked, you won’t send the _paq.push but instead you will send a message to the parent window',
```
Comment (prk3) [inline templates.js:456]:
```suggestion
          description: 'If checked, you will pass messages to the parent window instead of _paq.push',
```

**Review by prk3 — APPROVED** [review]

File: templates.js (hunk 1 above)
Comment (RomanSek) [reply templates.js:456]: Sounds good, but I copied description from other templates so I would have to change them too.
@kbigos What do you think? Should we stick with previous description or should we change them to this one?

Comment (kbigos) [reply templates.js:456]: Yeah I guess we can change them, it sounds better.

**Pull request merged**